packages/components/psammead-bulletin/package.json

```diff
@@ -21,7 +21,9 @@
   "dependencies": {
     "@bbc/gel-foundations": "^4.0.1",
     "@bbc/psammead-styles": "^4.3.0",
-    "@bbc/psammead-assets": "^2.13.0"
+    "@bbc/psammead-assets": "^2.13.0",
+    "@bbc/psammead-visually-hidden-text": "^1.2.3",
+    "@bbc/psammead-story-promo": "^5.1.0"
```

Comment:
These dependencies are not included in `package-lock.json` for bulletin and Jenkins is complaining